roles/ironic-control/tasks/main.yml

```diff
@@ -49,7 +49,7 @@
   tags: db-migrate
 
 - name: sync ironic database
-  command: "{{ ironic.virtualenv }}/bin/ironic-dbsync upgrade"
+  command: "/usr/local/bin/ironic-dbsync upgrade"
```

Comment:
my issue is that Ironic should really be virtualenv-ified, using a literal path here is not an issue for the short term but we really should venv all of ironic; python global package namespace installations are really bad practice.